stories/button.story.jsx

```diff
@@ -4,6 +4,8 @@ import { storiesOf } from "@storybook/react";
 import { withInfo } from "@storybook/addon-info";
 import { action } from "@storybook/addon-actions";
 import Button from "../src/button/Button";
+import AnchorButton from "../src/button/AnchorButton";
```

Comment:
Yea this is the right choice, having it as two different components rather than passing in a prop to change it.